application/sitebuilder/build.py

```diff
@@ -62,6 +76,35 @@ def enforce_static_mode():
             clear_up(build_dir)
 
 
+def build_and_upload_error_pages(application):
```

Comment:
Make this a private method?

Reply:
Again, matches the convention inside the file. None of the other ones for the standard site builder are private, so keeping these as-is.